duradmin/src/main/webapp/jquery/dc/ext/jquery.dc.common.js

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * 
+ *
```

Comment:
For the sake of clarity, it would be better if formatting changes were added in a separate commit.

Reply:
But I leave that call to your discretion.